.github/workflows/netlify-periodic-build.yml

```diff
@@ -1,12 +1,12 @@
 name: Trigger netlify build
 on:
   schedule:
-  - cron: '30 3 * * *'
+  - cron: '0 */6 * * *'
```

Comment:
Reason for changing this: To improve the freshness of data on the KEPs page. We are changing the rebuild cadence from once a day to 4 times a day.

Reply:
@sftim -- added the reasoning here.

Reply:
Could you put the reasoning _in the file_ (so it's obvious even if you don't got lack to look for a comment on this PR)?

Reply:
Makes sense. Added a comment to the workflow file.